docs/operators/jdbcPollInputOperator.md

```diff
@@ -0,0 +1,155 @@
+JDBC Poller Input Operator
+=============
+
+## Operator Objective
+This operator scans jdbc table in parallel fashion. The jdbc table is table in jdbc complaint database. To provide parallelism during read each instance of Apex input operator also known as operator partition reads one portion of jdbc table. Based on requested operator instances number of non-polling static partitions are created to scan table portitions. The non-polling partition will scan the table portion only once. Additional single operator partition is created to scan newly added data incrementally. 
+
+#### Assumption
+Assumption is that there is an ordered column using which range queries can be formed. That means database has a column which has unique constraint as well as the newly inserted record should have column value more than max value in that column.
+
+## Use cases
+1. Scan huge database tables to either copy to other database or process it using **Apache Apex**. The example application to copy database contents to HDFS is available in [examples repository](https://github.com/DataTorrent/examples/tree/master/tutorials/jdbcIngest)
+
+## How to Use?
+The tuple type in the abstract class is a generic parameter. Concrete subclasses need to choose an appropriate class (such as String or java class for [POJO](https://en.wikipedia.org/wiki/Plain_Old_Java_Object) etc for the generic parameter) and also implement a couple of abstract methods: getTuple(ResultSet) to parse database row to chosen class and emitTuple(T) to emit the tuple.
```

Comment:
Providing a link for POJO seems like overkill. Suggest removing reference to POJO and just say
"an appropriate concrete class". Also add a line stating that the concrete class should have a default (no-arg)
constructor so that it can be serialized by Kryo.
"parse database row to chosen class" => "convert database rows to objects of this concrete class"